src/components/Navbar/Navbar.tsx

```diff
@@ -47,38 +49,36 @@ export const Navbar = () => {
   return (
     <Box sx={{ flexGrow: 1 }}>
       <AppBar position="static">
-        <Toolbar>
-          <Grid2
-            container
-            width="100%"
-            direction="row"
-            textAlign={isMobile ? 'right' : 'left'}
-            spacing={1}
-          >
-            <Grid2 size={{ xs: 7, md: 4 }}>
-              <IconButton size="large" edge="start" color="inherit">
-                <Image
-                  src={LogoImage}
-                  height={isMobile ? 40 : 56}
-                  width={isMobile ? 40 : 56}
-                  alt="Logo for Koalaty Speech and Feeding."
-                />
-              </IconButton>
-            </Grid2>
+        <Toolbar disableGutters={isMobile}>
+          <Grid container width="100%" direction="row" spacing={1}>
             {!isMobile && (
-              <Grid2 size={4} alignContent="center">
-                <Typography
-                  variant="h6"
-                  textAlign="center"
-                  color="white"
-                  px={2}
-                >
-                  Koalaty Speech and Feeding Therapy Services
-                </Typography>
-              </Grid2>
+              <Grid size={{ md: 4 }}>
+                <IconButton size="large" edge="start" color="inherit">
+                  <Image
+                    src={LogoImage}
+                    height={56}
+                    width={56}
+                    alt="Logo for Koalaty Speech and Feeding."
+                  />
+                </IconButton>
+              </Grid>
             )}
-            <Grid2
-              size={{ xs: 5, md: 4 }}
+
+            <Grid size={{ xs: 11, md: 4 }} alignContent={'center'}>
+              <Typography
```

Comment:
The text is too big for the length of characters. We can default it to `subtitle1`:
```suggestion
  <Typography
      variant='subtitle1'
      textAlign={isMobile ? 'left' : 'center'}
      color={theme.palette.background.default}
      px={2}
  >
```

Reply:
Can you post a screen shot of this? now it seems to small in desktop mode.

Reply:
The purpose is to just make the font smaller since the text is very long. I would say try using a different font size if it's too small then

Reply:
We can review this later when we add an actual logo text here.